portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java

```diff
@@ -97,22 +94,12 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
       return null;
     }
 
-    // Making sure the file is an image file.
-    if (!"image".equals(blobInfo.getContentType().substring(0, 5))) {
+    // Ensure that uploaded file is an image file.
```

Comment:
This comment is unnecessary in its current form. Instead, I recommend removing it or updating it to explain why you are deleting the blobKey. Example: "Non-image files are not supported."

Reply:
That makes sense. Fixed.